metabase/metabase
# remove old unused static line area bar charts

### Description

https://github.com/metabase/metabase/pull/34820 removes the BE for the old static combo chart, this PR removes the FE part.

### How to verify

Send test subscriptions with static line/area/bar/combo charts and ensure they return a placeholder. You can verify this in Percy.

### Checklist

- [n/a] Tests have been added/updated to cover changes in this PR
